package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.9.3",
   "main": "src/index.jsx",
   "scripts": {
+    "deploy": "env HUSKY_SKIP_HOOKS=1 git-directory-deploy --directory build/ --branch=${DEPLOY_BRANCH:-build} --repo=${DEPLOY_REPOSITORY:-origin}",
```

Comment:
`deploy` est utilisé par Travis ?

Reply:
No it is used for adhoc deployments when you want to deploy a version of your app on a git branch.